src/components/__tests__/MontlyDataTab-test.tsx

```diff
@@ -0,0 +1,100 @@
+import { expect } from "chai";
+import * as Sinon from "sinon";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+import MonthlyDataTab from "../MonthlyDataTab";
+import CopyButton from "../CopyButton";
+import StatsInnerList from "../StatsInnerList";
+import DropdownButton from "../DropdownButton";
+import { Button } from "library-simplified-reusable-components";
+import { LibraryData } from "../../interfaces";
+import { testLibrary1, modifyLibrary, validate } from "./TestUtils";
+import { findYear } from "../../utils/sharedFunctions";
+
+describe("MonthlyDataTab", () => {
+  let data: {[key: string]: LibraryData[]};
+  let wrapper: Enzyme.CommonWrapper<any, any, {}>;
+  const today = new Date() as any;
+  const thisMonth = today.toLocaleString("default", { month: "long" });
+  const thisYear = findYear(today.toDateString())[0];
```

Comment:
From the tests, this looks fine but I've found that using the actual result from the Date object can cause problems with time and timezones in Travis or other services. In general, if possible, it'll be good to mock the Date object value so it is always the same value.

Reply:
Sounds like a good idea, but I'm not clear on what you mean; do you mean hard-code a specific date (which I'd rather not do, since much of the point of the component is that, on load, it automatically shows you data for whatever the current month and year are...)?  Or...?

Reply:
Yes, hard code the date just for the tests. You can override the Date object with `sinon` and comment that for the purposes of the test the date is hard-coded to a specific date but in real life it would display the current month and year. There's no issue right now but I'd keep a tab on this in case it causes errors running tests in the future.